doc/arduino.txt

```diff
@@ -110,6 +111,13 @@ Search these patterns to find a likely serial port to upload to. >
                                     \'/dev/tty.usbserial*']
 <
 
+                                               *'g:arduino_user_installation'*
+Set this variable to 1 if you want to use the the arduino user folder normally
+located under ~/.arduino15.
+If you're using a non default path you also have to set 'g:arduino_dir'. >
+  let g:arduino_user_installation = 0
+ 
+
```

Comment:
I believe you'll want a closing `<` here

Reply:
Done